docs/Course Wiki/MATH Course Offerings/MATH257.md

```diff
@@ -0,0 +1,54 @@
+
+## Content Covered
+
+- Linear equations, linear systems
+- Matrix operations, LU decomposition
+- Vector spaces, subspaces
+- Basis and dimension
+- Linear transformations
+- Eigenvalues, eigenvectors
+- Inner products and norms
+- Orthogonality, orthogonal projections, linear regression
+- Equilibrium, linear dynamical systems
+- Gram-Schmidt method, QR decomposition
+- Singular value decomposition, pseudo-inverse
+
+MATH257 is the first linear algebra course that cover many fundamental concepts in linear algebra. It starts with linear equations and linear systems, and introducing the concept of vectors and matrices to solve linear systems. The course then delves into matrix operations such as addition and multiplication, and the different types of matrices such as elementary and inverse matrices. All of these concepts are regularly utilized as you proceed onto more difficult topics such as subspaces, eigenvectors, linear transformations, and orthogonal projections, so make sure you are comfortable with the fundamentals.
+
+## Prerequisites
+ - [MATH 221](MATH221.md)
+
+
+## When to take it
+
+Students should take it whenever available after they have gained credit for [MATH221](MATH221.md), but most students take it after and while taking MATH241. The course is a prerequisite for [ECE313](../ECE%20Course%20Offerings/ECE313.md). 
+
+## Course structure
+Every week there are online lecture videos, two in-person lectures, one discussion, and one online lab. Generally, you start off watching the online lecture videos to learn new concepts, and then go to in-person lectures to learn more in-depth about those new concepts. Dicussions are in-person, where you work in groups to complete a dicussion worksheet. The lab is online on Zoom every Friday, where you work by yourself or with other students to complete Python programming assignments on Prairelearn/Jupiter Notebook. 
+
+The course has homework assigned weekly on Prairlearn, where you practice the concepts learned in lecture. The time to finish each homework varies, but most of them take about 1-2 hours. The homeworks give you unlimited attempts to get full credit, but most questions requires you to correctly answer 2-3 different variants to obtain full credit. 
+
+There are four midterms and one final exam in the course. Each midterm is 50 minutes long with about 12-13 questions, consisting of conceptual, computational questions, and one Python question based on the labs. It is recommended to review homework problems and tests from MATh 415, but unfortunately there are no practice midterms directly related to the actual midterms. The final exam is similar to the midterm, except that it is twice the length and 3 hours long. If your final exam scores is higher than any of your midterm scores, it will replaced your loweset midterm score.
```

Comment:
I forget if these are CBTF midterms / final, if so mention it.

Reply:
```suggestion
There are four midterms and one final exam in the course. Each midterm is 50 minutes long with about 12-13 questions, consisting of conceptual, computational questions, and one Python question based on the labs. It is recommended to review homework problems and tests from MATH 415 (previous version of MATH 257), but unfortunately there are no practice midterms directly related to the actual midterms. The final exam is similar to the midterm, except that it is twice the length and 3 hours long. If your final exam score is higher than any of your midterm scores, it will replace your lowest midterm score.
```